### PR TITLE
Fix #10703: Remove Lockwise page and add redirect

### LINKED ISSUE #10703: Updates to firefox/lockwise/
## Description

firefox/lockwise/ requires some changes to go live on 12/13/2021

Changes outlined here: https://docs.google.com/document/d/1Purpi1Rm_hk4RK_KjmOmT4XbxZxQz1ka31F5tBIcnk4/edit?usp=sharing

---

#### Success Criteria
- [ ] Code page
- [ ] changes go live on 12/13

### PR DESCRIPTION
**Pull request by pmac**

### CODE REVIEW

**Review by craigcook — APPROVED** [review]
I only found two lingering URL references that should be changed to `firefox.features.password-manager`:
https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/templates/firefox/browsers/compare/brave.html#L95
https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/templates/firefox/browsers/compare/safari.html#L94

There's a function in `media/js/firefox/browsers-products.js` that looks for a Lockwise button that is no longer there, but the JS fails gracefully so it's not breaking anything and we can clean that up some other time.

r+wc! 🧼

**Comment by pmac** [comment]
bah! editor didn't save. updated.

File: bedrock/firefox/templates/firefox/browsers/compare/ie.html [hunk 1]
```diff
@@ -89,7 +89,7 @@ <h2>{{ ftl('compare-shared-security-and-privacy-strong') }}</h2>
 
       <img src="{{ static('img/firefox/compare/compare-privacy-locks.svg') }}" alt="" height="217" width="320" class="compare-details-image">
 
-      <p>{{ ftl('compare-ie-so-whats-the-solution-if-your', fallback='compare-ie-so-whats-the-solution-if-your-fallback', lockwise='href="%s" data-cta-type="link" data-cta-name="Lockwise"'|safe|format(url('firefox.products.lockwise')),
+      <p>{{ ftl('compare-ie-so-whats-the-solution-if-your', fallback='compare-ie-so-whats-the-solution-if-your-fallback', lockwise='href="%s" data-cta-type="link" data-cta-name="Lockwise"'|safe|format(url('firefox.features.password-manager')),
```
Comment (robhudson) [inline bedrock/firefox/templates/firefox/browsers/compare/ie.html:92]: You changed the cta-name in the other two but not this one

Comment (pmac) [reply bedrock/firefox/templates/firefox/browsers/compare/ie.html:92]: good catch. thanks.

**Pull request merged**